nodes/vector_node.py

```diff
@@ -25,11 +25,14 @@
 
     else:
         async_robot = anki_vector.AsyncRobot()
-        async_robot.camera.init_camera_feed()
+        # async_robot.camera.init_camera_feed()
 
     # connect to Vector
     async_robot.connect()
 
+    # Camera feed?
+    async_robot.camera.init_camera_feed()
```

Comment:
Here, the client was not able to connect with the robot, it said that the client was doing something else and blocks after this.